src/main/charts/confluence/templates/statefulset.yaml

```diff
@@ -71,6 +65,8 @@ spec:
             - name: ATL_TOMCAT_SECURE
               value: "true"
             {{ end }}
+            - name: ATL_TOMCAT_ACCESS_LOG
+              value: {{ .Values.confluence.accessLog.enabled | quote }}
```

Comment:
The Confluence Docker image allows the access log to be turned on/off, so I've added that knob.